Tests/Marketplace/upload_packs.py

```diff
@@ -284,6 +339,7 @@ def option_handler():
                         action='store_true', required=False)
     parser.add_argument('-k', '--key_string', help="Base64 encoded signature key used for signing packs.",
                         required=False)
+    parser.add_argument('-P', '--private_bucket_name', help="Private storage bucket name", required=False)
```

Comment:
Bad name `-P`. Additionally, where in ci yml you pass `--private_bucket_name` to script?

Reply:
Couldn't think of a better letter for that... And I haven't added that to the config because the server doesn't support encrypted packs yet

Reply:
It is failing without it with following error: `ValueError: Cannot determine path without bucket name`. `private_storage_bucket` will be None and `download_and_extract_index` will raise exception. You need to skip this task in case no private bucket name was supported. `-p` for public and `-P` for private is confusing, better to remove `-P` shortcut at all.

Reply:
Ignore my comment regarding letter...

Reply:
Let's decide what to do with the config in the daily/afterwards